src/platform-apps/channels/channel-view.tsx

```diff
@@ -172,6 +175,10 @@ export class ChannelView extends React.Component<Properties, State> {
     );
   }
 
+  searchMentionableUsers = async (search: string) => {
```

Comment:
this method is in 2 places, what do you think if we move it to `message-input`, and let last be the responsible of it?

I can see only one reason why is here which because we need 2 data `id` and `users` and it's not changing nothing on this component - a thought

Reply:
`message-input` is a general input component but `channel-view` and `message` are context specific. If anyone else uses the `message-input` they'd want different ways to fetch the actual list of users they can mention in that context.

I can reduce the duplication by sending the function directly to the `message` component from the top level `channel-view` though. As far as I know, there's no reason for the `message` to fetch possible mentions differently.